document/core/binary/modules.rst

```diff
@@ -522,30 +524,19 @@ The :math:`\MEMORYINIT` and :math:`\DATADROP` instructions can only be used if t
        \MMEMS~\mem^\ast, \\
        \MGLOBALS~\global^\ast, \\
        \MELEM~\elem^\ast, \\
-       \MDATA~\data^{\X{m'}}, \\
+       \MDATA~\data^m, \\
        \MSTART~\start^?, \\
        \MIMPORTS~\import^\ast, \\
        \MEXPORTS~\export^\ast ~\} \\
-      \end{array} \\ &&&
-     (\begin{array}[t]{@{}c@{~}l@{}}
-      \iff & m^? \neq \epsilon \\
-      \vee & \forall (t^\ast, e) \in \X{code}^n, \MEMORYINIT \notin e \wedge \DATADROP \notin e) \\
-      \end{array} \\
+       \end{array} \\ &&&
+     (\iff m^? \neq \epsilon \vee \F{dataidx}(\X{code}^n) = \emptyset) \\
```

Comment:
Can we link this `\F` set to the definition below?

Reply:
Done.